src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let result = rent * days;
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
+  }
```

Comment:
Prefer using `if` with `return` over `if else` to simplify the conditions. After an `if` with a `return` statement, you can directly return the default case without using `else`.